routes/api.php

```diff
@@ -33,13 +33,14 @@
     'index', 'show'
 ]);
 
-
 Route::group(['prefix' => 'user', 'middleware' => 'auth:sanctum'], function () {
     Route::get("", [UserController::class, 'index']);
     Route::get("/{id}", [UserController::class, 'show']);
 });
 
 Route::get('venues/{id}/reviews', [VenueController::class, 'get_reviews']);
+Route::get('/venues/suggest/shortest', [VenueController::class, 'suggest_shortest_path']);
+//Route::get('/venues/suggest/all', [VenueController::class, 'suggest_all']);
```

Comment:
Im ok with leaving this here for now.

Reply:
Hmm I would argue if we are not using it then get rid. You can always add comments or a little bit in the documentation as to practices around naming convention??